### PR TITLE
build(release): fix releases not updating package.json's version field

### PR DESCRIPTION
**Pull request by necipallef**

### CODE REVIEW

**Review by Orkuncakilkaya — APPROVED** [review]

File: .releaserc [hunk 1]
```diff
@@ -100,6 +100,18 @@
...
     ],
     "@semantic-release/changelog",
+    [
+      "@semantic-release/npm",
+      {
+        "npmPublish": false
+      }
+    ],
+    [
+      "@semantic-release/exec",
```
Comment (ilfa) [inline .releaserc:110]: You need to install the package

Comment (necipallef) [reply .releaserc:110]: Done. Could you re-review? @ilfa

Comment (necipallef) [reply .releaserc:110]: We also need to release `v1.0.1`. Do you want me to create an empty commit? I can also change commit messages with `fix` prefix, and force push. @ilfa

Comment (ilfa) [reply .releaserc:110]: let's rewrite the commit message

**Review by ilfa — APPROVED** [review]

**Pull request merged**